scripts/docker-run-rootfs.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -e
```

Comment:
As you're using this, you should probably consider some kind of cleanups, there is bunch of steps which might fail leaving unnecessary garbage in the system.

Reply:
I removed it.